tests/Main.hs

```diff
@@ -35,7 +35,7 @@ main = do
     describe "ide" TestIde.spec
     beforeAll TestUtils.setupSupportModules $ do
       describe "compiler" TestCompiler.spec
-      describe "bundle" TestBundle.spec
+      -- describe "bundle" TestBundle.spec
```

Comment:
Should this comment be deleted altogether?

Reply:
Yeah, it should.